core/shared/src/main/scala/zio/prelude/Traversable.scala

```diff
@@ -269,11 +270,25 @@ trait Traversable[F[+_]] extends Covariant[F] {
 
 object Traversable extends LawfulF.Covariant[DeriveEqualTraversable, Equal] {
 
+// need to implement natural transformations for this property?
+// val naturalityLaw: LawsF.Covariant[DeriveEqualTraversable, Equal] = ???
```

Comment:
I think we can restate this as a fusion law that if we have `F[G[H[A]]]` where `F` is traversable and `G` and `H` have `IdentityBoth` and `Covariant` then we can individually `flip` twice or we can create a composed instance and `flip` the whole thing once.

Reply:
Thanks for the big tip!